### PR TITLE
43-6-Apna_Time_Ayega

### PR DESCRIPTION
**Pull request by mynk322**

### CODE REVIEW

File: Leaderboard/After Task-1.md [hunk 1]
```diff
@@ -56,7 +56,7 @@
   </tr>
   <tr>   
       <tr>
- <td>43</td><td>6</td><td>Apna_Time_Ayega</td><td>50</td><td><a href="https://github.com/amankumarkeshu">Aman Kumar</a></td><td></td><td></td>
+ <td>43</td><td>6</td><td>Apna_Time_Ayega</td><td>50</td><td><a href="https://github.com/amankumarkeshu">Aman Kumar</a></td><a href="https://github.com/mynk322">MAYANK PADIA</a><td></td><td></td>
   </tr>
```
Comment (sarthakgupta072) [inline Leaderboard/After Task-1.md:60]: Insert your name in b/w the "td" tag.  :P

**Review by sarthakgupta072 — CHANGES_REQUESTED** [review]

**Comment by sarthakgupta072** [comment]
Note: You could have done the changes in the earlier PR too. There was no need to open a separate PR for every change you make.

Close either one of the pull requests.

**Comment by mynk322** [comment]
hi

**Comment by sarthakgupta072** [comment]
I am closing the other pull request. Make changes in this PR only.

**Pull request closed without merging**